### ISSUE TITLE
Not working for CLI installed via NPM

### ISSUE THREAD

**Issue by JasperV** [blacs30/bitwarden-alfred-workflow#11]
Hi,

I had some troubles getting the workflow running. Seems that because I have installed the BW CLI via NPM (and NVM), that the workflow cannot find it.

I have tinkered with the code a bit;

```python
def get_bw_exec():
    cmd = "which bw"
    proc = Popen(cmd.split(), env=my_env, stdout=PIPE, stderr=PIPE)
    bw_exec, err = proc.communicate()
    if "not found" in err.decode('utf8').strip():
        print(json.dumps(error_result(type="install")))
        exit(2)
    return bw_exec
```

This seems to fix it really quickly. I also noticed some other hard references to `/usr/local/bin/bw`.
Not sure if those are used. My best bet would be that in those cases it would also fail to work.

Python is not really my game, so I am unsure to put up a pull-request with some changes.

Would really appreciate for this to be fixed.

**Comment by blacs30** [comment]
Thanks a lot for reporting this.
That is very helpful I believe.
I've only used `brew` to install the bitwarden cli but not npm.

I agree that this should be fixed and should be more flexible.

**Comment by blacs30** [comment]
This was fixed with release [2.0.0](https://github.com/blacs30/bitwarden-alfred-workflow/releases/tag/2.0.0) 

The PATH env var and the location of the Bitwarden cli executable can be configured now.

**Comment by JasperV** [comment]
Awesome!
